### PR TITLE
Added "enter" as a button option for quick restart

### PR DESCRIPTION
**Pull request by SHaiderX**
Added "enter" as a button option for quick restart. I feel like the enter key is super convenient to press after a round to quick start. I saw myself pressing it after each round because it felt intuitive, but realised it wasnt an option. 

I added the change in every possible file I could find that mentioned "quick restart". Please let me know if I missed anywhere or if I need to add anything else in order for it to work.

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4409545195

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4415498120

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4415534136

**Comment by Miodec** [comment]
There is a logic issue, also from my limited testing Enter doesn't work to restart

File: frontend/src/ts/commandline/index.ts [hunk 1]
```diff
@@ -454,7 +454,9 @@ $(document).ready(() => {
         (event.key?.toLowerCase() === "p" &&
           (event.metaKey || event.ctrlKey) &&
           event.shiftKey) ||
-        ((event.key === "Tab" || event.key === "Escape") &&
+        ((event.key === "Tab" ||
+          event.key === "Escape" ||
+          event.key === "enter") &&
```
Comment (Ferotiq) [inline frontend/src/ts/commandline/index.ts:459]: This is likely the reason for it not working, `enter` I think should be `Enter`

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4429096750

**Comment by Miodec** [comment]
@SHaiderX are you coding this blind? Enter is still not doing anything

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4456968018

File: frontend/src/ts/controllers/input-controller.ts [hunk 2]
```diff
@@ -702,7 +702,10 @@ function handleTab(event: JQuery.KeyDownEvent, popupVisible: boolean): void {
       setWordsInput(" " + TestInput.input.current);
       return;
     }
-  } else if (Config.quickRestart === "tab") {
+  } else if (
+    Config.quickRestart === "tab" ||
+    Config.quickRestart === "escape"
```
Comment (Ferotiq) [inline frontend/src/ts/controllers/input-controller.ts:707]: Here is your issue, "escape" is 1. not a quickRestart option ("esc") and 2. is probably meant to be "enter" here.

If you are making a change to the frontend logic, please make sure you test this logic.

**Comment by Miodec** [comment]
Closing due to inactivity

**Pull request closed without merging**